### PR TITLE
Merge stable into master

### PR DESCRIPTION
**Pull request by MoonlightSentinel**

### CODE REVIEW

**Comment by ibuclaw** [comment]
Is master<->stable merge doing well? There seems to be more here than what there should be (again, for the third time now since ~2.098.0 merge).

**Comment by MoonlightSentinel** [comment]
The diff looks correct but the PR lists many commits whose changes are already in master.
IIRC the previous merge PR's were rebased? That might explain the empty commits (same change but different commit hash)

**Comment by ibuclaw** [comment]
> The diff looks correct but the PR lists many commits whose changes are already in master. IIRC the previous merge PR's were rebased? That might explain the empty commits (same changes but different commit hash)

I did a merge rebase for 2.098.0->master, but then just a straightforward merge was done for 2.098.1->master.  This should not be happening again AFAIU.

File: src/dmd/typesem.d [hunk 1]
```diff
@@ -4654,7 +4654,7 @@ Expression dotExp(Type mt, Scope* sc, Expression e, Identifier ident, int flag)
  * Returns:
  *  The initialization expression for the type.
  */
-extern (C++) Expression defaultInit(Type mt, const ref Loc loc, const bool isCfile = false)
+Expression defaultInit(Type mt, const ref Loc loc, const bool isCfile = false)
```
Comment (ibuclaw) [inline src/dmd/typesem.d:4657]: Keep it `extern (C++)`.

**Pull request closed without merging**